configuration/src/main/java/io/airlift/configuration/ConfigurationModule.java

```diff
@@ -17,14 +17,29 @@
 
 import com.google.inject.Binder;
 import com.google.inject.Module;
+import com.google.inject.binder.AnnotatedBindingBuilder;
 
 import java.lang.annotation.Annotation;
+import java.util.stream.Stream;
 
 import static io.airlift.configuration.ConfigBinder.configBinder;
 
 public class ConfigurationModule
         implements Module
 {
+    public static ConfigurationAwareModule installModules(Module... modules)
```

Comment:
Let's move this below, next to the other static method. We normally only put static factories for the current class at the top. Having methods here distracts from the class fields.